feat: allow to opt-out of use:enhance for forms (#13250)
sveltejs/kit: .changeset/spotty-queens-beam.md

---
'@sveltejs/kit': minor
---

> Allow to do opt out of enhancing forms with `data-sveltekit-reload` on the form element or button

Reviewer: ```suggestion
feat: allow opting out of `use:enhance` with `data-sveltekit-reload` on the form element or submit button
```
